### PR TITLE
added demo app example cards to maps sdk example page

### PR DESCRIPTION
**Pull request by langsmith**
This pr adds example cards to https://www.mapbox.com/android-docs/map-sdk/examples to bring the page up-to-date with the examples that are already living in [the android demo app](https://github.com/mapbox/mapbox-android-demo).

Snapshot:
screen shot 2018-06-18 at 10 59 00 am
DDS:
screen shot 2018-06-18 at 10 58 54 am

Styles:
screen shot 2018-06-18 at 10 58 45 am

### CODE REVIEW

File: src/pages/map-sdk/examples/index.js [hunk 1]
```diff
@@ -321,6 +351,41 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
             exampleImgID="examplesStyleMapHotspotsRuntime"
             exampleUrl="https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/CreateHotspotsActivity.java"
           />
+          <ExampleCard
+            exampleTitle="Heat map data"
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: In the [style spec](https://www.mapbox.com/mapbox-gl-js/style-spec#layers-heatmap) heatmap is one word. We should use that here, too, for consistency.

File: src/pages/map-sdk/examples/index.js [hunk 2]
```diff
@@ -321,6 +351,41 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
...
+            exampleDescription="Add and customize a heat map to visualize data."
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: `heat map` --> `heatmap`

File: src/pages/map-sdk/examples/index.js [hunk 3]
```diff
@@ -321,6 +351,41 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
...
+            difficulty="intro"
+            exampleImgID="examplesDdsHeatmapData"
+            exampleUrl="https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/HeatmapActivity.java"
+          />
+          <ExampleCard
+            exampleTitle="Styling heat maps"
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: `heat map` --> `heatmap`

File: src/pages/map-sdk/examples/index.js [hunk 4]
```diff
@@ -321,6 +351,41 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
...
+            exampleDescription="Gain minute control over heat map coloring, size, density, and other styling."
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: `heat map` --> `heatmap`

File: src/pages/map-sdk/examples/index.js [hunk 5]
```diff
@@ -380,6 +445,24 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
             exampleUrl="https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/query/QueryFeatureActivity.java"
           />
         </ExampleCardContainer>
+        <ExampleCardContainer exampleTitle="Image generation">
+          <ExampleCard
+            exampleTitle="Snapshot notification"
+            exampleDescription={
+              'Show a snapshotted map image in a notification'
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: Add a `.` to the end of the description.

File: src/pages/map-sdk/examples/index.js [hunk 6]
```diff
@@ -380,6 +445,24 @@ let MapSdkExamplesLayout = class MapSdkExamplesLayout extends React.Component {
...
+            }
+            difficulty="intro"
+            exampleImgID="examplesSnapshotNotification"
+            exampleUrl="https://github.com/mapbox/mapbox-android-demo/blob/master/MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/snapshot/SnapshotNotificationActivity.java"
+          />
+          <ExampleCard
+            exampleTitle="Snapshot sharing"
+            exampleDescription={'Send and share a map snapshot image'}
```
Comment (colleenmcginnis) [inline src/pages/map-sdk/examples/index.js]: Add a `.` to the end of the description.

Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

File: src/pages/map-sdk/examples/index.js (hunk 5 above)
Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

File: src/pages/map-sdk/examples/index.js (hunk 4 above)
Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

File: src/pages/map-sdk/examples/index.js (hunk 3 above)
Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

File: src/pages/map-sdk/examples/index.js (hunk 2 above)
Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

File: src/pages/map-sdk/examples/index.js (hunk 1 above)
Comment (langsmith) [reply src/pages/map-sdk/examples/index.js]: Fixed

**Pull request merged**